h/cli/commands/move_uri.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+import click
+
+from h import models
+from h.api import uri
+from h.api.search.index import BatchIndexer
+
+
+@click.command()
```

Comment:
I have a fanatical dislike for underscores on the command line, sorry. Can we make this `@click.command('move-uri')`?